paimon-core/src/main/java/org/apache/paimon/manifest/ManifestCommittable.java

```diff
@@ -72,6 +84,10 @@ public void addLogOffset(int bucket, long offset, boolean allowDuplicate) {
         logOffsets.put(bucket, newOffset);
     }
 
+    public void addProperty(String key, String value) {
```

Comment:
Can you show the example? How to add property? Add what property.

Reply:
Maybe something like:
```
ManifestCommittable committable = new ManifestCommittable(COMMIT_IDENTIFIER);
committable. addProperty("fluss_offset_bucket1", "14")
committable. addProperty("fluss_offset_bucket2", "15")
```